infrastructure/terraform/modules/backend-api/kms_key_s3.tf

```diff
@@ -0,0 +1,5 @@
+resource "aws_kms_key" "s3" {
```

Comment:
I've been wondering if we get any benefit from having so many kms keys. Would it not be sufficient to just use the key in `infrastructure/terraform/components/app/module_kms.tf`? Should probably see what Aiden thinks

Reply:
I know this is just the way we've always done it from the core project, maybe not the place to pull you up on this. 

My thinking is that KMS costs money and the data that we keep in SQS, S3, DynamoDB etc is all the same data. I'm not sure what benefit we get from having a different key per service, we could just have one key for the whole application right?

If we needed other keys, it feels like it should be on grounds of like data security - e.g. use a more restricted key for encrypting PII data.